src/apps/survey/demoSurveyConfig.ts

```diff
@@ -114,13 +114,15 @@ export const config: ISurveyConfig = {
       },
     ],
     cesiumContext: {
+      cesiumApiKey:
+        'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIxMWU4YzQzNC00NzMxLTQ0NzktYTFlYi01NjMyMDgwMTMyY2EiLCJpZCI6MjI2NjUyLCJpYXQiOjE3MzkxODcxNTZ9.OJJ_pdI3WDMLO3W4vYWA1aW20DilQ2nyocgItAWPs-g',
```

Comment:
We can define a `defaultAccessToken` if you want.

Reply:
![image](https://github.com/user-attachments/assets/a3613ebf-3b09-4ba7-8ca8-198537144dc4)

it is done already

Reply:
Let's rename it so that it matches Cesium naming.